www/source/partials/docs/_dev-pkgs-share.html.md.erb

```diff
@@ -53,7 +53,7 @@ Once Builder possesses at least the public key of the origin, you may upload one
 
 <%= partial "/partials/global/channel-overview" %>
 
-By default, newly uploaded packages are placed in the `unstable` channel, however, the default package that is downloaded is the latest `stable` version of a package, unless overridden in commands such as `hab start`, `hab svc load`, and `hab install`. If you want to promote your package to the `stable` channel, run the `hab pkg promote` command as follows:
+By default, newly uploaded packages are placed in the `unstable` channel, however, the default package that is downloaded is the latest `stable` version of a package, unless overridden in commands such as `hab svc start`, `hab svc load`, and `hab install`. If you want to promote your package to the `stable` channel, run the `hab pkg promote` command as follows:
```

Comment:
This needs to be fixed. See https://www.habitat.sh/docs/using-builder/#sharing-pkgs